src/index.test.js

```diff
@@ -106,16 +106,15 @@ const match = {
   url: '/finc-select',
 };
 
-const renderWithRouter = (component) => {
-  const history = createMemoryHistory();
-  return {
-    ...render(withIntlConfiguration(
-      <Router history={history}>
-        {component}
-      </Router>
-    ))
-  };
-};
+const history = createMemoryHistory();
+
+const renderWithRouter = (component) => (
+  renderWithIntlConfiguration(
+    <MemoryRouter history={history}>
+      {component}
+    </MemoryRouter>
+  )
+);
```

Comment:
```suggestion

const renderWithRouter = (component) =>
  renderWithIntlConfiguration(<MemoryRouter>{component}</MemoryRouter>);
```

`<MemoryRouter>` takes care of the history.